### PR TITLE
feat: add context to rate-limited event

### PR DESCRIPTION
**Pull request by levenleven**
###  Summary

Add api request url and body to `WebClientEvent.RATE_LIMITED` event parameters

Resolves https://github.com/slackapi/node-slack-sdk/issues/1636

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Review by seratch — APPROVED** [review]
This enhancement makes a lot of sense and it's not a breaking chance. Since this PR contains a document change, we should merge right before releasing the next minor version

**Comment by seratch** [comment]
The CI build error needs to be resolved before merging this PR https://github.com/slackapi/node-slack-sdk/issues/1638

**Review by filmaj — APPROVED** [review]
Looks good, and thanks for the tests!

**Comment by filmaj** [comment]
I am going to cut a release of this package now.

**Pull request merged**
